src/components/Login.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { motion } from "framer-motion";
+import classes from "./Login.module.css";
+import logo from "../images/logo.png";
+import {Link} from 'react-router-dom'
+
+const Login = () => {
+  return (
+    <motion.div
```

Comment:
I would extract `motion.div` into common component and you could receive dynamic props into those components